community-operators/radanalytics-spark/sparkcluster.crd.yaml

```diff
@@ -0,0 +1,13 @@
+kind: CustomResourceDefinition
+apiVersion: apiextensions.k8s.io/v1beta1
+metadata:
+  name: sparkclusters.radanalytics.io
+spec:
+  group: radanalytics.io
+  names:
+    kind: sparkcluster
```

Comment:
I'm not sure why, but these CRs still show up as `parkcluster` in the UI.
@alecmerdler Do you have any insight into this?

Reply:
It also happens with the other CRD as well: `parkapplication`

Reply:
note: in the next version of spark operator, the CRDs will be `CamelCased` (https://github.com/radanalyticsio/spark-operator/pull/184)